index2.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles2.css">
+    <title>Concept pagina</title>
+</head>
+
+<body>
+<header class="outer-container">
+
+    <nav class="navbar">
+        <ul>
+            <li><a href="#rolling-stone-title">case studies</a></li>
+            <li><a href="#webdesign-title">services</a></li>
+            <li><a href="#footer-text">contact us</a></li>
+        </ul>
+    </nav>
+
+    <div class="hero-section">
+        <h1>Your website is an investment
+            in the succes of your business.</h1>
+        <p>We are here to help you succeed.</p>
+        <button class="cta.btn" type="button">Lets work together</button>
+    </div>
+
+
+    <img src="assets/squiggle_line_stripe.svg" alt="My Happy SVG" class="swirl">
+    <div class="skewed">skewed</div>
+</header>
+
+
+<main class="main1">
+
```

Comment:
Tevens kon je de <main> omwikkelen van regel 33 tot 182 en vervolgens de tag <sections> gebruiken. Zo klopt de opbouw van de pagina.

Reply:
dat is een goeie, ik had het inderdaad beter in kunnen delen in <main> en vervolgens <section1> <section2>.

thanks!